lib/check.js

```diff
@@ -57,6 +57,7 @@ module.exports = function (options, callback) {
   }
 
   options.proxy = options.proxy || Conf.proxy;
+  options.proxy = options.proxy || process.env.https_proxy || process.env.HTTPS_PROXY;
```

Comment:
Why not just join this line with the previous line, and use something like this?

```js
options.proxy = options.proxy || Conf.proxy || process.env.https_proxy || process.env.HTTPS_PROXY;
```